packages/jest-resolve-dependencies/src/index.ts

```diff
@@ -129,8 +155,9 @@ class DependencyResolver {
     }
     const modules: Array<DependencyResolver.ResolvedModule> = [];
     for (const file of this._hasteFS.getAbsoluteFileIterator()) {
+      const res = this.resolve(file, options);
```

Comment:
unnecessary diff

Reply:
Agreed, this from the original PR. I'll remove it :+1: